docker-compose.yml

```diff
@@ -1,16 +1,30 @@
 version: "3.9"
 
 services:
-  tests:
-    image: composer:2.1
+  tests-7.4:
```

Comment:
Add new Dockfiles to our docker-compose manifest for more robust containerized tests, and support for 8.1 on devices where executables may not be ready for testing yet.